src/types/gecko-profile.js

```diff
@@ -120,10 +120,10 @@ export type GeckoFrameStruct = {
 
 export type GeckoStackTable = {
   schema: {
-    frame: 0,
-    prefix: 1,
+    prefix: 0,
+    frame: 1,
   },
-  data: Array<[IndexIntoGeckoFrameTable, IndexIntoGeckoStackTable | null]>,
+  data: Array<[IndexIntoGeckoStackTable | null, IndexIntoGeckoFrameTable]>,
```

Comment:
I added a reference to `docs-developer/gecko-profile-format.md` at the top of the types file to better explain how these types come about. Luckily these types were created after the code was written, so the initial code was correct, but my types were not.

I'd like to keep the `gecko-profile-format.md` file as the canonical reference to document the gecko profile format since it's liable to break, and will need updating.